docs/_includes/sidenav/mobile.html

```diff
@@ -1,14 +1,14 @@
 {% assign current = page.url | downcase | split: '/' %}
 
-<aside class="sidenav-mobile">
-  <a class="sliding-panel-close" href="javascript:void(0);">
-    <img src="/assets/img/close.svg" alt="close">
-  </a>
-  <nav>
+<nav role="navigation" class="usa-nav sidenav-mobile">
```

Comment:
Do these changes relate to the header or a solve for another issue? No big deal, but want to make sure to account for any changes that help solve any open issues we need to account for.